Core/fft_mt_r2iq.cpp

```diff
@@ -189,36 +189,31 @@ void fft_mt_r2iq::Init(float gain, int16_t **buffers, float** obuffers)
 		float *pht = new float[halfFft / 4 + 1];
 		for (int d = 0; d < NDECIDX; d++)
 		{
-			const float stopdb = 120.0f;
 			switch (d)
 			{
 			case 5:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 0.8f/64.0f, 1.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 80.0f, 0.7f/64.0f, 1.0f/64.0f, pht);
 				break;
 			case 4:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 1.8f/64.0f, 2.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 90.0f, 1.6f/64.0f, 2.0f/64.0f, pht);
 				break;
 			case 3:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 3.8f/64.0f, 4.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 100.0f, 3.6f/64.0f, 4.0f/64.0f, pht);
 				break;
 			case 2:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 7.8f/64.0f, 8.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 110.0f, 7.6f/64.0f, 8.0f/64.0f, pht);
 				break;
 			case 1:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 15.75f/64.0f, 16.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 120.0f, 15.7f/64.0f, 16.0f/64.0f, pht);
 				break;
 			case 0:
 			default:
-				KaiserWindow(halfFft / 4 + 1, stopdb, 30.5f/64.0f, 32.0f/64.0f, pht);
+				KaiserWindow(halfFft / 4 + 1, 120.0f, 30.5f/64.0f, 32.0f/64.0f, pht);
 				break;
 			}
 			for (int t = 0; t < (halfFft/4+1); t++)
 			{
-#ifdef WIDEFFTN
-				pfilterht[t][0] = pfilterht[t][1] = ( pht[t] / sqrtf(2.0f) ) / 4.0f ;  // gain adj
-#else
-				pfilterht[t][0] = pfilterht[t][1] = pht[t] / sqrtf(2.0f);
-#endif
+				pfilterht[t][0] = pfilterht[t][1] = ( pht[t] / sqrtf(2.0f) ) / ((float)FFTN_R_ADC/2048);  // gain adj reference was 2048
```

Comment:
can we also consider GainScale here? So we can skip one multiply over the result?

Reply:
YES. I will add it.